modules/scan/scan.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * The entry point to the admin bar notice class.
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Scan;
+
+require_once 'class-admin-bar-notice.php';
+
+Admin_Bar_Notice::instance();
```

Comment:
I like having the 2 things separate. So that including a file doesn't have any side effects. Such as creating an instance of a class. 

Will update the comment to be more generic though.